crowns.lic

```diff
@@ -1,5 +1,5 @@
 =begin
-  Documentation: https://elanthipedia.play.net/Lich_script_repository#dice-game
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#Hollow_Eve_Dice_Game
```

Comment:
💡 this is usually the name of the script, i.e. `crowns`

Reply:
I will change it around. I put the script name in the header of it. Wanted to make sure everyone knew was HE dice and not some dice game elsewhere.